### PR TITLE
fix: [PIE-5783]: Added pipelineBranchName from URL query params in Pipeline Input step for triggers for new git sync pipeline

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by pankajsingh-harness** [comment]
retrigger titlecheck

**Comment by pankajsingh-harness** [comment]
retrigger jest

**Review by rameshx — APPROVED** [review]

File: src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx [hunk 1]
```diff
@@ -1404,7 +1404,7 @@ const TriggersWizardPage: React.FC = (): JSX.Element => {
         originalPipeline,
         resolvedPipeline,
         inputSetTemplateYamlObj,
-        pipelineBranchName: getDefaultPipelineReferenceBranch(triggerTypeOnNew),
+        pipelineBranchName: getDefaultPipelineReferenceBranch(triggerTypeOnNew) || branch,
```
Comment (samarth-sharma31) [inline src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx:1407]: Can we add this default branch to this util itself -> getDefaultPipelineReferenceBranch

Comment (pankajsingh-harness) [reply src/modules/72-triggers/pages/triggers/TriggersWizardPage.tsx:1407]: @samarth-sharma31, Triggers are still in refactoring and getDefaultPipelineReferenceBranch is used at many other places. 
I will check this while doing more refactoring.

**Review by samarth-sharma31 — APPROVED** [review]

**Pull request merged**